src/xb-builder.c

```diff
@@ -435,7 +435,7 @@ static void
 xb_builder_nodetab_write_sentinel (XbBuilderNodetabHelper *helper)
 {
 	XbSiloNode sn = {
-		.is_node	= FALSE,
+		.flags		= XB_SILO_NODE_FLAG_NONE,
```

Comment:
Nitpick: Would have been a bit easier to review this if the changes to add accessor functions were split out into separate commits, but it’s too late now.